### PR TITLE
Feat update documentation files

### LINKED ISSUE #3: Create Readme, Contributing.md and CONDUCT.md.

### PR DESCRIPTION
**Pull request by mel-liow**
Taken from milestone 1:

README.md:
_Outline the package you would like to build in the README.md file. In particular, your README.md should contain:_
- [x] A summary paragraph that describes the project at a high level
- [x] A bulleted list of the functions (and datasets if applicable) that will be included in the package (this should be a 1-2 sentence description for each function/dataset)
- [x] a paragraph describing where your packages fit into the Python ecosystem (are there any other Python packages that have the same/similar functionality? Provide links to any that do. If none exist, then clearly state this as well).

CONTRUBUTORS.md:
- [x] edit CONTRIBUTORS.md to add the contributor names (the students on the teams).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
 # AirPyllution
+A package for visualizing future, historic and current air pollution data using the [OpenWeather API](https://openweathermap.org).
```
Comment (christopheralex) [inline README.md]: Is it only about visualization ? Since we have a function or two which doesn't return a graph.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+- `getPollutionHistory()`
+- `getPollutionForecast()`
+
+### `getAirPollution()`
+Fetches the air pollution levels based on a location (string). Based on the values of the polluting gases, this package uses the [Air Quality Index](https://en.wikipedia.org/wiki/Air_quality_index#CAQI) to determine the level of pollution for the location and produces a coloured map of the area displaying the varying regions of air quality.
+
+### `getPollutionHistory()`
+Requires a start and end date and fetches historic air pollution data for a specific location. The function returns a data frame with the values of the polluting gases over the specified date range.
+
+### `getPollutionForecast()`
```
Comment (christopheralex) [inline README.md:25]: Does this function mean we are solving a predictive problem ? Or is this data present in the dataset we generate ?

File: README.md (hunk 1 above)
Comment (mel-liow) [reply README.md]: Nope that's true, I can change it to be:
"A package for visualising or obtaining future, historic, and current air pollution data using the [OpenWeather API](https://openweathermap.org)."

File: README.md (hunk 2 above)
Comment (mel-liow) [reply README.md:25]: No, we're simply just returning the data fetched from the API as a dataframe. 
The API response for the Historical air pollution request (https://openweathermap.org/api/air-pollution) returns data in a JSON so we could create a function that turns the JSON object into a table?

**Comment by mel-liow** [comment]
@danfke @christopheralex  Just an FYI - the descriptions are not set in stone - happy to change the function description/ functionality later once we're done with the POC

File: README.md (hunk 2 above)
Comment (christopheralex) [reply README.md:25]: Sorry Mel , should I have mentioned the function. Im asking about `getPollutionForecast()` ?

File: README.md (hunk 1 above)
Comment (christopheralex) [reply README.md]: Perfect 👍

File: README.md (hunk 2 above)
Comment (christopheralex) [reply README.md:25]: > @danfke @christopheralex Just an FYI - the descriptions are not set in stone - happy to change the function description/ functionality later once we're done with the POC

But as mentioned. We can go ahead and merge it as these are our preliminary assumptions/functionalities.

Comment (mel-liow) [reply README.md:25]: Oh i see, sorry! The forecast data is also part of the API - `Forecast air pollution data` They provide us with forecasted data for the next 5 days

Comment (christopheralex) [reply README.md:25]: Cool 👍

**Pull request merged**